__tests__/Export-Excel-TotalSettings.Tests.ps1

```diff
@@ -0,0 +1,71 @@
+﻿#Requires -Modules @{ ModuleName="Pester"; ModuleVersion="4.0.0" }
```

Comment:
My Windows 10 machine had Pester 3.4.0 **pre-installed** which was unable to perform the tests. Turns out you need at least Pester 4.